HQSmokeTests/testPages/users/org_structure_page.py

```diff
@@ -196,3 +205,24 @@ def cleanup_location(self):
         self.click(self.delete_org_level)
         self.wait_to_click(self.save_btn_delete)
         print("Org level deleted successfully")
+
+    def archive_location(self):
+        self.wait_to_click(self.org_menu_link_text)
+        active_loc = self.get_text(self.archive_this_loc)
+        self.wait_to_click(self.archive_buttton)
+        self.wait_to_click(self.archive_button_popup)
+        self.is_present_and_displayed(self.archive_success_message)
+        self.wait_to_click(self.show_arhcived_locations)
+        self.is_present_and_displayed(self.archive_this_loc)
+        archived_loc = self.get_text(self.archive_this_loc)
+        assert archived_loc == active_loc, "Location is not Archived"
+
+    def unarchive_location(self):
+        self.wait_to_click(self.org_menu_link_text)
+        self.wait_to_click(self.show_arhcived_locations)
+        archived_loc =self.get_text(self.archive_this_loc)
+        self.wait_to_click(self.unarchive_button)
+        self.wait_to_click(self.show_active_locations)
+        self.is_present_and_displayed(self.archive_this_loc)
+        unarchived_loc = self.get_text(self.archive_this_loc)
+        assert unarchived_loc == archived_loc, "Location not unarchived successfully"
```

Comment:
Same, we should pass when vice-versa of [this](https://github.com/dimagi/dimagi-qa/pull/109/files#r897706390) is true.